api/server.py

```diff
@@ -225,7 +225,9 @@ def login():
     '''
     User tries to login via google and is redirected back to / if successful.
     '''
-    if not google.authorized:
+    if session.get('email') in whitelist:
+        return jsonify({"success": True})
```

Comment:
We should redirect to `/` here. This route is opened as a new target in the browser, it won't be called from javascript, so the user would see this success json without a redirect. The js app will use `/api/login_status` to check if the user is in or not.